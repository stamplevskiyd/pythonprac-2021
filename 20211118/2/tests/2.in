with open ("tests/2.in") as f:
    txt = f.read()
    class C:
        num = Num()
        
    c = C()
    c.num = txt
    print(c.num)    
