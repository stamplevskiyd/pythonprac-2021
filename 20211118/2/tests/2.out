0
True
10
